modules/@apostrophecms/doc-type/index.js

```diff
@@ -328,6 +328,15 @@ module.exports = {
           }
         }
       },
+      afterDelete: {
+        async deleteRelatedReverseId(req, doc) {
+          // When deleting an unlocalized or draft document,
+          // we remove related reverse IDs of documents having a relation to the deleted one
+          if (!doc.aposMode || doc.aposMode === 'draft') {
```

Comment:
unlocalized docs have no `doc.aposMode`, should be enough? I also added a test for unlocalized pieces to be sure it works.

Reply:
Yes.